Candidate: CVE-2024-50636
PublicDate: 2024-11-11 23:15:00 UTC
References:
 https://www.cve.org/CVERecord?id=CVE-2024-50636
 https://github.com/schrodinger/pymol-open-source/issues/405
 https://github.com/yamerooo123/CVE/blob/main/CVE-2024-50636/Description.md
 https://youtu.be/SWnN_a1tUNc
Description:
 PyMOL 2.5.0 contains a vulnerability in its "Run Script" function, which
 allows the execution of arbitrary Python code embedded within .PYM files.
 Attackers can craft a malicious .PYM file containing a Python reverse shell
 payload and exploit the function to achieve Remote Command Execution (RCE).
 This vulnerability arises because PyMOL treats .PYM files as Python scripts
 without properly validating or restricting the commands within the script,
 enabling attackers to run unauthorized commands in the context of the user
 running the application.
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
Priority: medium
Discovered-by:
Assigned-to:
CVSS:
 cisa-adp: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H [9.8 CRITICAL]

Patches_pymol:
upstream_pymol: needs-triage
esm-apps/xenial_pymol: needs-triage
esm-apps/bionic_pymol: needs-triage
focal_pymol: ignored (end of standard support, was needs-triage)
esm-apps/focal_pymol: needs-triage
jammy_pymol: needs-triage
esm-apps/jammy_pymol: needs-triage
noble_pymol: needs-triage
esm-apps/noble_pymol: needs-triage
oracular_pymol: ignored (end of life, was needs-triage)
plucky_pymol: needs-triage
questing_pymol: needs-triage
devel_pymol: needs-triage
